### ISSUE TITLE
Mute toggle

### ISSUE THREAD

**Issue by DroidFloyd** [deckerst/aves#399]
Please make the mute toggle save state for all following videos.
•When you switch videos the state changes back to mute.

**Comment by deckerst** [comment]
I agree with your suggestion, as long as you stay on the viewer page.

But if you leave the viewer, back to your collection, and reopen the viewer, you don't expect the mute state to be preserved, right?

**Comment by DroidFloyd** [comment]
Thanks for replying. No preferably it would go back to mute after leaving.

**Comment by DroidFloyd** [comment]
> Please make the mute toggle save state for all following videos.
> •When you switch videos the state changes back to mute.

•Please if you could also add saved state for the speed adjuster as well. Sorry for the bother
